### PR TITLE
Export props to allow other libraries to reference it

### PR DESCRIPTION
**Pull request by labithiotis**
Pretty simple PR to export props to allow them to be reference by other libraries.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
 } from './helpers'
 import cssStyles from './styles.css'
 
-type Props = {
+export type EasyCropperProps = {
```
Comment (ValentinH) [inline src/index.tsx:15]: As the component is named Cropper, I would name this CropperProps.

Comment (labithiotis) [reply src/index.tsx:15]: @ValentinH Can do, but thought due to NPM name it would be best to relate to that i.e `react-easy-crop` therefor 'EasyCropper`

Comment (ValentinH) [reply src/index.tsx:15]: Cropper is the name used in the docs so I'd use this :)

**Comment by ValentinH** [comment]
Thanks for this PR. It has been published under v3.2.1 🎉

**Comment by ValentinH** [comment]
@all-contributors please add @labithiotis for code

**Pull request merged**